recipe/meta.yaml

```diff
@@ -31,9 +29,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - johanneskoester
-    - bgruening
-    - daler
-    - jdblischak
-    - cbrueffer
-    - dbast
+    - conda-forge/r
```

Comment:
This makes it recursive. You need the members explicitly listed here.

Reply:
Oh, right. Will fix.